flink-kubernetes-operator/pom.xml

```diff
@@ -270,6 +283,13 @@ under the License.
                             <version>${flink.version}</version>
                             <outputDirectory>${plugins.tmp.dir}/flink-metrics-statsd</outputDirectory>
                         </artifactItem>
+                        <artifactItem>
```

Comment:
It is strange to only add the oss filesystem here. We need to bundle all the internal fs implementation or none.

Reply:
As discussed in the mail list, I think we should avoid bundle all the fs jars It will make the image bigger and not necessary. I intend to add here as an example for the user to follow up the guide, But I think I can remove it now and clarify how to extend it in the doc.

Reply:
Removed

Reply:
Yes. We could remove it now and document it with `kustomize`.

Reply:
Get it. I will add it in the followup doc ticket

Reply:
We just could add the document in FLINK-27270.

Reply:
Yes